From e4707bfb24a8175018975a542f6e1c3b84ac02dd Mon Sep 17 00:00:00 2001
From: Alexandre Rostovtsev <tetromino@gentoo.org>
Date: Sun, 21 Oct 2012 00:26:24 -0400
Subject: [PATCH] test: disable test_fail_abstract_new

Depending on how glib is configured, whether there are systemwide
debugging tools running, etc., this test can produce other signals, not
necessarily signal 11.
---
 tests/check/gst/gstobject.c | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/tests/check/gst/gstobject.c b/tests/check/gst/gstobject.c
index d5956db..25995bf 100644
--- a/tests/check/gst/gstobject.c
+++ b/tests/check/gst/gstobject.c
@@ -531,15 +531,6 @@ gst_object_suite (void)
 
   /* FIXME: GLib shouldn't crash here, but issue a warning and return a NULL
    * object, or at least g_error() and then abort properly ... (tpm) */
-#ifndef HAVE_OSX
-  /* Disabled for OS/X because a) it's a pretty silly test anyway and
-   * b) different OS/X versions raise different signals and it isn't worth
-   * the effort to try and detect which one should be producing which
-   */
-  /* SEGV tests go last so we can debug the others */
-  if (g_getenv ("CK_FORK") == NULL || strcmp (g_getenv ("CK_FORK"), "no") != 0)
-    tcase_add_test_raise_signal (tc_chain, test_fail_abstract_new, SIGSEGV);
-#endif
 
   return s;
 }
-- 
1.7.12.3

